src/test/java/uk/gov/pay/api/utils/mocks/CreateChargeResponseFromConnector.java

```diff
@@ -0,0 +1,176 @@
+package uk.gov.pay.api.utils.mocks;
+
+import uk.gov.pay.api.model.CardDetails;
+import uk.gov.pay.api.model.PaymentState;
+import uk.gov.pay.api.model.RefundSummary;
+import uk.gov.pay.api.model.SettlementSummary;
+import uk.gov.pay.commons.model.SupportedLanguage;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Optional;
+
+public class CreateChargeResponseFromConnector {
+    public final Long amount, corporateCardSurcharge, totalAmount; 
+    public final PaymentState state; 
+    public final String chargeId, returnUrl, description, reference, email, paymentProvider, gatewayTransactionId, createdDate;
```

Comment:
Might be a question for Alex and the style guide, I thought we favour one member per line, although I know you're a fan of this more concise way.

Reply:
Yeah I don't remember us discussing this ever... I think as long as it's readable and understandable its fine.

Reply:
OK.